server/services/reportedAdjudicationsService.test.ts

```diff
@@ -138,7 +141,7 @@ describe('reportedAdjudicationsService', () => {
     })
 
     it('returns the correct data', async () => {
-      const result = await service.getYourCompletedAdjudications(user, {
+      const result = await service.getYourCompletedAdjudications(user, null, {
```

Comment:
I think we need a test for the filters as well (but that can be added in another PR)

Reply:
Yeah you are correct - I figure I'll put this in when doing the report adjudications stuff.